src/tagify.scss

```diff
@@ -186,7 +192,16 @@
 
     ///////////////////////////////////////////
     // Holds the placeholder & the tags input
-    &__input{ display:inline-block; min-width:110px; margin:$tagMargin; padding:$tagPad; position:relative; vertical-align:top;
+    &__input{
+        /* NOTE(LordMike): when using 'inline-block' 
+            Edge will remove whole <div contenteditable> when all content text is selected and delete is pressed.
+            We cannot use 'inline-block' here. For now.
+        */
+    	/* display:inline-block; */ 
+    	display:block;
```

Comment:
If the parent already has `display:flex`  this means the children do not need any `display` property of themselves, since they are flex children